### PR TITLE
Update 'Message' description

### PR DESCRIPTION
**Pull request by brettasmi**
This updates the `components/schemas/Message` description ahead of the 1.0 release.

### CODE REVIEW

**Review by edeutsch — CHANGES_REQUESTED** [review]
Would be good to mention the KnowledgeGraph in the description. The three things are the QueryGraph, the KnowledgeGraph, and the Results (which are the answers that bind the KG nodes and edges to QG nodes and edges)

**Comment by CaseyTa** [comment]
In the knowledge_graph description, I think we've removed support for remote knowledge graphs. Can anyone else confirm?

**Comment by patrickkwang** [comment]
> In the knowledge_graph description, I think we've removed support for remote knowledge graphs. Can anyone else confirm?

Confirmed.

File: TranslatorReasonerAPI.yaml [hunk 1]
```diff
@@ -131,11 +131,18 @@ components:
       required:
         - message
     Message:
+      description: >-
+        The message object holds the main content of a Query or a Response.
+        This includes the query configuration in the query_graph property,
+        any answers in the results property, and a mapping of response nodes
+        and edges to their QueryGraph counterparts in the knowledge_graph
+        property. The content of these properties is context-dependent
```
Comment (CaseyTa) [inline TranslatorReasonerAPI.yaml:139]: > any answers in the results property, and a mapping of response nodes
>         and edges to their QueryGraph counterparts in the knowledge_graph
>         property

This wording is a little confusing to me. It sounds like the results property and the mapping of response nodes and edges are separate things, when I believe you're referring to the same thing.

File: TranslatorReasonerAPI.yaml [hunk 2]
```diff
@@ -148,9 +157,8 @@ components:
         knowledge_graph:
           type: object
           description: >-
-            KnowledgeGraph object that contains all the nodes and edges
-            referenced in any of the possible answers to the query OR
-            connection information for a remote knowledge graph
+            KnowledgeGraph object that contains lists of nodes and edges
+            and edges in the thought graph corresponding to the message
```
Comment (CaseyTa) [inline TranslatorReasonerAPI.yaml:161]: There's an extra "and edges" here.

**Review by CaseyTa — CHANGES_REQUESTED** [review]
There's an extra "and edges" in the description for `knowledge_graph`. 

Otherwise, I approve of these changes.

**Review by CaseyTa — APPROVED** [review]

**Review by edeutsch — APPROVED** [review]

**Review by patrickkwang — APPROVED** [review]

**Review by vdancik — APPROVED** [review]

**Review by rtroper — APPROVED** [review]

**Comment by brettasmi** [comment]
Thanks for the feedback everyone. I think I need @edeutsch, @patrickkwang or someone else with privileges to merge this.

**Pull request merged**